src/main/resources/org/jenkinsci/plugins/nomad/NomadCloud/config.jelly

```diff
@@ -26,16 +51,10 @@
     <f:textbox default="1"/>
   </f:entry>
 
-  <f:entry title="Nomad ACL" field="nomadACLCredentialsId" description="Valid Nomad ACL Token">
-    <c:select/>
-  </f:entry>
-
   <f:entry title="Jenkins Worker URL" field="workerUrl" description="Jenkins worker URL">
     <f:textbox default="${instance.getWorkerUrl()}"/>
   </f:entry>
 
-  <f:validateButton title="Test connection" progress="Testing API connectivity..." method="testConnection" with="nomadUrl"/>
-
```

Comment:
I moved "connection-related" things around to put them closer and have the "test connection" button closer to what it actually checks (I used it quite a while during the review...)